README.md

```diff
@@ -256,6 +256,27 @@ robot -v SERVER:localhost:5000 -v SUPERUSER_USERNAME:test@opev.net -v SUPERUSER_
 ```
 * Once the tests are completed, a report and a log would be generated at `report.html` and `log.html` respectively in your root directory.
 
+### Pre-commits guide
+
+**Introduction**
+
+Git hook scripts are useful for identifying simple issues before submission to code review. We run our hooks on every commit to automatically point out issues in code such as missing semicolons, trailing whitespace, and debug statements. By pointing these issues out before code review, this allows a code reviewer to focus on the architecture of a change while not wasting time with trivial style nitpicks.
+
+#### 1. Add a pre-commit configuration:
+
+* create a file named .pre-commit-config.yaml
```

Comment:
https://github.com/fossasia/open-event-server/pull/6991/files/bfca5cf0ce60f786e40479630e3a758801b183f3#r426085332

Only include the install command and link to config.